### PR TITLE
Add myself to ios list

### PR DESCRIPTION
**Pull request by thargett-babylon**

### CODE REVIEW

**Review by danspndl — CHANGES_REQUESTED** [review]
Please follow the steps at the beginning of the table (update [scripts/squad.yml](https://github.com/Babylonpartners/ios-playbook/blob/master/scripts/squads.yml) then run [squads.rb](https://github.com/Babylonpartners/ios-playbook/blob/master/scripts/squads.rb)) (#225)

File: README.md [hunk 1]
```diff
@@ -101,6 +101,7 @@ The rest of the iOS Engineers work in the following squads:
       <td>Chesley Stephens</td><td><a href='https://github.com/cstephens-babylon'>@cstephens-babylon</a></td><td></td></tr>
       <tr><td>Patrick Westmeyer</td><td><a href='https://github.com/bh-pwestmeyer'>@bh-pwestmeyer</a></td><td></td></tr>
       <tr><td>Sam Francis</td><td><a href='https://github.com/SamFrancis-Babylon'>@SamFrancis-Babylon</a></td><td></td></tr>
+      <tr><td>Tyler Hargett</td><td><a href='https://github.com/thargett-babylon'>@thargett-babylon</a></td><td></td></tr>
```
Comment (mluisbrown) [inline README.md]: There's something off with the column alignment:

image

You probably need to increase the `rowspan` at line 100 to 4.

**Review by mluisbrown — DISMISSED** [review]
Small formatting issue.

**Comment by mluisbrown** [comment]
Ok, looks like @sdaniel55 has the proper solution 😄

**Review by danspndl — APPROVED** [review]
Welcome to the team 🚀

**Review by Daniel1of1 — APPROVED** [review]
👋 Welcome

**Pull request merged**